pkg/codegen/internal/test/testdata/aws-fargate.pp.ts

```diff
@@ -0,0 +1,97 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+
+const vpc = aws.ec2.getVpc({
+    "default": true,
+});
+const subnets = vpc.then(vpc => aws.ec2.getSubnetIds({
+    vpcId: vpc.id,
+}));
+// Create a security group that permits HTTP ingress and unrestricted egress.
+const webSecurityGroup = new aws.ec2.SecurityGroup("webSecurityGroup", {
+    vpcId: vpc.id,
+    egress: [{
+        protocol: "-1",
+        fromPort: 0,
+        toPort: 0,
+        cidrBlocks: ["0.0.0.0/0"],
+    }],
+    ingress: [{
+        protocol: "tcp",
+        fromPort: 80,
+        toPort: 80,
+        cidrBlocks: ["0.0.0.0/0"],
+    }],
+});
+// Create an ECS cluster to run a container-based service.
+const cluster = new aws.ecs.Cluster("cluster", {});
+// Create an IAM role that can be used by our service's task.
+const taskExecRole = new aws.iam.Role("taskExecRole", {assumeRolePolicy: {
+    Version: "2008-10-17",
+    Statement: [{
+        Sid: "",
+        Effect: "Allow",
+        Principal: {
+            Service: "ecs-tasks.amazonaws.com",
+        },
+        Action: "sts:AssumeRole",
+    }],
+}});
+const taskExecRolePolicyAttachment = new aws.iam.RolePolicyAttachment("taskExecRolePolicyAttachment", {
+    role: taskExecRole.name,
+    policyArn: "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy",
+});
+// Create a load balancer to listen for HTTP traffic on port 80.
+const webLoadBalancer = new aws.elasticloadbalancingv2.LoadBalancer("webLoadBalancer", {
+    subnets: subnets.ids,
```

Comment:
This doesn’t work does it?

Reply:
Hm, it does not. I'll have to file a bug :)